pkg/storage/wrapper/prefix/prefix_wrapper.go

```diff
@@ -37,20 +36,20 @@ type StorePrefixWrapper struct {
 
 // Put stores v with k ID by prefixing it with IDPrefix.
 func (b *StorePrefixWrapper) Put(k string, v []byte) error {
-	if k == "" {
-		return fmt.Errorf("cannot Put with empty key")
+	if k != "" {
```

Comment:
@Baha-sk @fqutishat

Reply:
We should not check it here. We need to rely on DB.

Reply:
I agree

Reply:
sure, it's ok then